src/TestSuite/IntegrationTestCase.php

```diff
@@ -343,11 +367,11 @@ protected function _buildRequest($url, $method, $data)
         ];
         $session = Session::create($sessionConfig);
         $session->write($this->_session);
-
         list ($url, $query) = $this->_url($url);
+
         $props = [
             'url' => $url,
-            'post' => $data,
+            'post' => $this->_addTokens($url, $method, $data),
```

Comment:
This will add the security token conditionally but adds the `_csrfToken` in any case or I overlooked something? What happens if the user doesn't use `CsrfComponent`? The `_csrfToken` itself doesn't hurt but it will change the request data.